### ISSUE TITLE
[Enhancement] Affix functionality for kirby-input

### ISSUE THREAD

**Issue by tspringborg** [kirbydesign/designsystem#2368]
## Describe the enhancement
Increasingly there's a use for prefix and suffix elements on kirby-input.

Specific usecases: 
Adding `kr` or similar as a suffix 
Adding the calculated age in years as a suffix.
Adding a loading indicator as a prefix when something is loading.

## Describe the solution you'd like
Preferably add 2 more @Input's to the kirby-input directive. Suffix input and prefix input.
The input should take a TemplateRef or a simple string. The simple string would should a default styled text while a TemplateRef would meet all other needs.

## Have you considered any alternatives?
I have made a directive for our own project but preferably it should be a part of the kirby-input directive.

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement
- [x] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by jkaltoft** [comment]
Closing issue. Duplicate of https://github.com/kirbydesign/designsystem/issues/459
